AsyncDisplayKit/Details/ASEnvironment.m

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import "ASEnvironment.h"
+
+ASEnvironmentLayoutOptionsState ASEnvironmentLayoutOptionsStateCreate()
+{
+  return (ASEnvironmentLayoutOptionsState) {
+    .spacingBefore = 0,
```

Comment:
I removed the default values but didn't remove the creation functions in case we want to have default values in the future.